### PR TITLE
feat: add framework logos where available

### PR DESCRIPTION
**Pull request by ericapisani**
🎉 Thanks for submitting a pull request! 🎉

#### Summary
Builds on #797 

Includes the logos for most of the frameworks.

Some things to note:

Generally been opting for the icons themselves, but there's some with the full name of the framework in the logo if we'd rather use those:
* Astro
* Blitz
* Quasar
* Qwik
* Svelte / Svelte-Kit
* Stencil

Missing logos:
* Docpad
* next-nx - not sure what logo we want to use for this since this is a combination of Nx and Next
* Zola - it looks like they don't have an image, but just CSS styles applied to 'Zola'

Other notes:
* Docusaurus v1 and v2 use the same logo.
* Nuxt and Nuxt 3 use the same logo.
* Could only find 1 logo for Grunt. They had something about more assets in their docs but I wasn't able to find anything
* Harp has a really light logo that I think will be difficult to see in a light theme. Might need to reach out about getting a different kind of logo, unless we want to but the logo itself on a background to make it visible
* Assemble is going to [provide an SVG](https://github.com/assemble/assemble/issues/967), have a PNG as a placeholder
* Want the Parcel one double checked - it's missing the box but it might be good enough for now
* React static logo is only available in PNG. I can ask about an SVG but the project is in maintenance mode so not sure about whether it's worth it to ask
* Svelte and Svelte kit use the same logo. Kit has some changes to their branding based on https://github.com/sveltejs/branding/issues/6 so am using what they have available at the moment.

**A picture of a cute animal (not mandatory, but encouraged)**

IMG_20220730_164508_733

### CODE REVIEW

File: src/frameworks/angular.json
Comment (MarcL) [inline src/frameworks/angular.json:22]: [dust] Do we want to only support the production domain? We could add the deploy preview if necessary (but maybe too much extra work for this and unlikely to be needed?)

Comment (ericapisani) [reply src/frameworks/angular.json:22]: As an MVP I was just going to add the production domain since it made for slightly less testing on my part, but I can take a look at making this change while I wait for reviews 😄

Comment (ericapisani) [reply src/frameworks/angular.json:22]: Have added it to the base branch of this PR in [this commit](https://github.com/netlify/framework-info/pull/797/commits/20f8e7c410d81229279d4fe45bce6b634ab9070f), so assuming that's a good approach I can update the paths here as well to support non-production domains

File: src/frameworks/stencil.json
Comment (nickytonline) [inline src/frameworks/stencil.json:20]: Are there SVG versions of this logo available?

File: src/frameworks/react-static.json
Comment (nickytonline) [inline src/frameworks/react-static.json:20]: Are there SVG versions of this logo?

**Review by nickytonline — APPROVED** [review]
Looks great. Just wondering about some assets that are PNGs. 🚀

**Comment by nickytonline** [comment]
> Missing logos:
> * Docpad
> * next-nx - not sure what logo we want to use for this since this is a combination of Nx and Next
> * Zola - it looks like they don't have an image, but just CSS styles applied to 'Zola'

Can you create a separate issue for the above?

> * Could only find 1 logo for Grunt. They had something about more assets in their docs but I wasn't able to find anything

> * Harp has a really light logo that I think will be difficult to see in a light theme. Might need to reach out about getting a different kind of logo, unless we want to but the logo itself on a background to make it visible

Definitely reach out to Harp

> * Assemble is going to [provide an SVG](https://github.com/assemble/assemble/issues/967), have a PNG as a placeholder

You can the above to the missing logos issue I suggested above.

> * Want the Parcel one double checked - it's missing the box but it might be good enough for now

You can create an issue in the Parcel repo asking about a full logo.

**Review by MarcL — APPROVED** [review]
Looking good. 🎉

File: src/frameworks/stencil.json
Comment (ericapisani) [reply src/frameworks/stencil.json:20]: There are, I just forgot to update this reference, so good catch

**Comment by ericapisani** [comment]
@nickytonline Sounds good, I'll start making those issues once this PR has been merged in

**Review by MarcL — APPROVED** [review]
All good. Thanks for the PR! 🎉

**Pull request merged**
